build-packages.cmd

```diff
@@ -1,18 +1,47 @@
 @if "%_echo%" neq "on" echo off
-setlocal
+setlocal EnableDelayedExpansion
 
 set packagesLog=build-packages.log
 echo Running build-packages.cmd %* > %packagesLog%
 
-if /I [%1] == [/?] goto Usage
+set native=false
+set options=/nologo /v:minimal /flp:v=detailed;Append;LogFile=%packagesLog%
+
+:Loop
+if [%1]==[] goto Begin
+
+if /I [%1]==[/?] goto Usage
 if /I [%1] == [/help] goto Usage
 
+if /I [%1] == [/n] (
+  set native=true
+  goto Next
+)
+
+if /I [%1] == [/native] (
+  set native=true
+  goto Next
+)
+
+echo Unrecognized argument '%1'
```

Comment:
Thinking about this I believe I would just eliminate the batch file argument completely and just delegate all the parameters to msbuild and to enable this we can pass `build-packages.cmd /p:SkipManagedPackageBuild=true`